KVMarketSquare/Favorites.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Favorites.swift
+//  KVMarketSquare
+//
+//  Created by Ka Hin Ng on 2021-12-13.
+//
+
+import Foundation
+
+class Favorites: ObservableObject {
```

Comment:
I think, we can rename this class as `AppData` or something general so we can have more data object inside of this class.